docs/medical-api/api-reference/fhir/create-patient-consolidated.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Create Patient's Consolidated Data"
+description: "Create Patient's Consolidated Data with a FHIR Bundle."
+api: "POST /medical/v1/patient/{id}/consolidated"
+---
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Body
+
+<ParamField body="resourceType" type="string" required>
+  The resource needs to be "Bundle"
+</ParamField>
+
+<ParamField body="type" type="string" required>
+  The type needs to be "collection"
+</ParamField>
+
+<ParamField body="entry" type="array" required>
+  The entry needs to be an array of [FHIR resources](/medical-api/fhir/resources).
+</ParamField>
+
+## Response
+
+Returns a [Bundle](/medical-api/fhir/resources/bundle) of type "transaction-response" with the entry being the outcome of the query.
+
+```json
+{
+  "resourceType": "Bundle",
+  "id": "b2c28be3-8aae-4b3c-b0ff-ca5d0f0b0d1e",
+  "type": "transaction-response",
+  "link": [
+    {
+      "relation": "self",
+      "url": "https://localhost:8888/oauth/fhir/21fa432e-723b-4a1d-a3b2-bd9cd75a0717/21fa432e-723b-4a1d-a3b2-bd9cd75a0717"
```

Comment:
I suggest replacing `localhost:8888` by a production domain - and making sure IDs don't exist there.